src/calculateRentalCost.js

```diff
@@ -3,8 +3,29 @@
  *
  * @return {number}
  */
+
+const DAY_COST_VALUE = 40;
+const THREE_DAYS_OFF_VALUE = 20;
+const SEVEN_DAYS_OFF_VALUE = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const isInvalidData = typeof days !== 'number';
+
+  if (isInvalidData) {
+    return 0;
+  }
```

Comment:
The function currently returns 0 for invalid data types, which could be misleading as 0 could also be a valid return value for 0 days. It would be better to throw an error to explicitly handle invalid input.